Makefile

```diff
@@ -32,11 +36,13 @@ $(TESTS):
 	# TODO: Figure out how to name multiple CC and produce corresponding object for each
 	$(CXX) $(SRC_FLAGS) $(GTEST_FLAGS) $(PARSER_PATH)config_parser.cc server_config.cc $(TESTS).cc ${GTEST_DIR}/src/gtest_main.cc libgtest.a $(LDFLAGS) -o $(TESTS)
 
-test: $(TARGET) $(TESTS)
+integration_test: $(TARGET) $(TESTS)
```

Comment:
Yeah, part of the reason behind the currently weird naming scheme is that I wasn't exactly sure how to/if it's possible for a target to call multiple targets. I'll make the issue for this.